tests/test_jmcomic/test_jm_api.py

```
@@ -1,7 +1,50 @@
+    def test_fetch_and_decrypt_images(self, jm_id: int = 438516, output_dir: str = "output_dir"):
+        """
+        测试下载并解密图片
+        :param jm_id: 本子的禁漫车号
+        :param output_dir: 输出目录
+        """
+        import os
+
+        # 确保输出目录存在
+        os.makedirs(output_dir, exist_ok=True)
+        # 获取专辑详情和图片数据列表
+        album, image_data_list = get_album_images(jm_id)
```

Ruff (0.8.2)

`get_album_images` may be undefined, or defined from star imports

(F405)